src/components/autocomplete-grid/autocomplete-grid/autocomplete-grid.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { expect } from '@open-wc/testing';
+import type { TemplateResult } from 'lit';
+import { html } from 'lit/static-html.js';
+
+import { isSafari } from '../../core/dom';
+import { describeIf } from '../../core/testing';
+import { fixture, testA11yTreeSnapshot } from '../../core/testing/private';
+
+import type { SbbAutocompleteGridElement } from './autocomplete-grid';
+import './autocomplete-grid';
+import '../autocomplete-grid-row';
+import '../autocomplete-grid-option';
+import '../autocomplete-grid-actions';
+import '../autocomplete-grid-button';
+
+describe('sbb-autocomplete-grid', () => {
+  let root: SbbAutocompleteGridElement;
+  const grid: TemplateResult = html`
+    <sbb-autocomplete-grid>
+      <sbb-autocomplete-grid-row>
+        <sbb-autocomplete-grid-option>Option 1</sbb-autocomplete-grid-option>
+        <sbb-autocomplete-grid-actions>
+          <sbb-autocomplete-grid-button icon-name="dog-small"></sbb-autocomplete-grid-button>
+        </sbb-autocomplete-grid-actions>
+      </sbb-autocomplete-grid-row>
+      <sbb-autocomplete-grid-row>
+        <sbb-autocomplete-grid-option>Option 2</sbb-autocomplete-grid-option>
+        <sbb-autocomplete-grid-actions>
+          <sbb-autocomplete-grid-button icon-name="dog-small"></sbb-autocomplete-grid-button>
+        </sbb-autocomplete-grid-actions>
+      </sbb-autocomplete-grid-row>
+    </sbb-autocomplete-grid>
+  `;
+
+  beforeEach(async () => {
+    root = await fixture(grid);
```

Comment:
```suggestion
    root = await fixture(html`...`);
```
Use the template directly